spec/operators/catch-spec.ts

```diff
@@ -242,4 +242,15 @@ describe('Observable.prototype.catch', () => {
       done();
     });
   });
+
+  it('should allow types with operators such as mergeMap to flow correctly', () => {
+    type(() => {
+      Observable.from([{ a: 1 }, { a: 2 }, { a: 3} ])
+          .mergeMap(value => {
+              return Observable.of([value]);
+          })
+          .mergeMap(x => x)
+          .subscribe(x => x.a);
+    });
+  });
```

Comment:
I'm not sure the best place to put it... but before the changes this code will fail.  With the changes this test will pass... the only reason it's in here is because the related change was made to catch.

(the `type` method just compiles the code, it doesn't actually run the test code)